Configure

```diff
@@ -469,6 +469,7 @@ my @disablables = (
     "rc4",
     "rc5",
     "rdrand",
+    "rndr",
```

Comment:
It might make sense for these two options to be combined into an `rdcpu` option.

Reply:
The changes for `INSTALL.md` show it part of rdcpu.  But it seems this is here so you can turn it off.

Reply:
The point was that rdrand and rndr are mutually exclusive and could be covered just as well by a single option.

Reply:
> The point was that rdrand and rndr are mutually exclusive and could be covered just as well by a single option.

Ah, in which case add a new CPU-agnostic option like "rdcpu" and make rdrand be an alias?

Reply:
Since the logic for `rndr` engine is being removed from this PR and only locations related to the `rndr` engine use the macro `OPENSSL_NO_RDRAND`, then this line will end up being removed too.